providers/default.rb

```diff
@@ -360,9 +360,15 @@ def execute_install_script(install_script)
     # sysvinit won't restart after we exit, potentially use cron to do so
     # either trust the chef-client cookbook's init scripts or the users choice
     if (node['chef_client'] && node['chef_client']['init_style'] == 'init') || node['chef_client_updater']['restart_chef_via_cron']
+      start_cmd = case node['platform_family']
+                  when 'aix'
+                    '/usr/bin/startsrc -s chef > /dev/console 2>&1'
+                  else
+                    '/etc/init.d/chef-client start'
+                  end
+
       r = cron 'chef_client_updater' do
-        minute '*/1'
```

Comment:
?

Reply:
Good question @tas50 

I elaborated on that in the PR comment:

Cron notes 📓

This PR removes the `minute` property from the cron job, since the original format of `minute '*/1'` is not compatible on all systems, such as AIX. This is OK because the `cron` cookbook sets `*` as the default for all [fields](https://github.com/chef-cookbooks/cron#properties).

Reply:
The resulting entry is `* * * * * /the/job`